tools/pd-ctl/pdctl/ctl.go

```diff
@@ -37,9 +38,10 @@ type CommandFlags struct {
 var (
 	commandFlags = CommandFlags{}
 
-	detach   bool
-	interact bool
-	version  bool
+	detach            bool
+	interact          bool
+	version           bool
+	readlinecompleter *readline.PrefixCompleter
```

Comment:
According to go naming convention https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps, it's better to use `readlineCompleter` instead of `readlinecompleter`. And the rest is the same.

Reply:
got it